ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -57,6 +57,21 @@
                     FormView.customFunctions.defaultSubmitErrorHandler(defaultSubmissionError, globals);
                 }
             }, "submitError");
+            this._model.subscribe((action) => {
```

Comment:
Have we checked that we don't need to override hardcoded save handlers like we have done for submit? Or we will provide it later if customer ask?

Reply:
As of now, we dont have any customer for this so we can do once some one asks for this.